pkg/lib/bundle/validate_test.go

```diff
@@ -16,17 +16,10 @@ func TestPullBundle(t *testing.T) {
 	tag := "quay.io/example/bundle:0.0.1"
 	dir := "/tmp/dir"
 
-	logger := logrus.NewEntry(logrus.New())
-
 	mockImgReader := containertoolsfakes.FakeImageReader{}
 	mockImgReader.GetImageDataReturns(nil)
 
-	validator := imageValidator{
-		imageReader: &mockImgReader,
-		logger:      logger,
-	}
-
-	err := validator.PullBundleImage(tag, dir)
+	err := mockImgReader.GetImageData(tag, dir)
```

Comment:
This is no longer testing anything.

Reply:
really? because I believe this change make it test exactly what is here.

Reply:
It looks like you're testing the output of a mock. If that's not the case, then I apologize.

Reply:
did I not put the link in. I'm sorry. GitHub recently glitches. (a lot of time I was shown not signed in) This is the link. I am just testing what was in the function. (btw, if you can not tell my tone, most of what I say are questions. Please feel free to correct me!)  https://github.com/operator-framework/operator-registry/blob/5e8da3d2e213abad0ca871cf873cdc7479c5e4e0/pkg/lib/bundle/validate.go#L51

Reply:
I think this was still testing exactly what it was. wdyt

Reply:
Almost. The subtle difference is that this was testing a **real** validator's pull function before, backing it with a mock. Now, this test is only using **mocks**, which is what I mean by "doesn't test anything".

Reply:
But I see your point -- since it's just a pass-through, that means this wasn't testing much to begin with. I would either get rid of the test, or make it more meaningful.

Reply:
Image pull needs and e2e environment and it is covered already. This test does not make sense because it is not pulling anything. I still think the error test case might be meaningful. but the first one is not really testing anything.